framework/src/modules/chain/chain.js

```diff
@@ -307,30 +307,27 @@ module.exports = class Chain {
 			}),
 			getLastBlock: async () => this.blocks.lastBlock,
 			blocks: async action => this.transport.blocks(action.params || {}),
-			getHighestCommonBlockId: async action => {
+			getHighestCommonBlock: async action => {
 				const valid = validator.validate(
-					definitions.getHighestCommonBlockIdRequest,
+					definitions.getHighestCommonBlockRequest,
 					action.params,
 				);
 
 				if (valid.length) {
 					const err = valid;
 					const error = `${err[0].message}: ${err[0].path}`;
-					this.logger.debug(
-						'getHighestCommonBlockId request validation failed',
-						{
-							err: error,
-							req: action.params,
-						},
-					);
+					this.logger.debug('getHighestCommonBlock request validation failed', {
```

Comment:
I think syntax is opposite.
It shoukd be 
debug({}, 'message')

Reply:
True, done